### PR TITLE
Migrating an on-prem web app to the cloud: ASP.NET on IIS to Azure App Services

### PR DESCRIPTION
**Pull request by robpearson**
# About

This is a post about Migrating an on-prem web app to the cloud: ASP.NET on IIS to Azure App Services. It's part of the Octopus 2021 Q2 launch materials. 

# Pre-requisites

- [x] The draft is complete and this post is ready to be reviewed.
- [x] I'm familiar with the [writing for Octopus TL;DR](https://style.octopus.com/writing-at-octopus-tldr).
- [x] The PR build passes validation, and if it doesn't, I've checked the [common validations errors](https://style.octopus.com/writing-at-octopus-tldr#common-validation-errors) and none of those apply.
- [x] The images I've included follow the [rules for working with images](https://style.octopus.com/images#rules-for-screenshots).

## How to review this PR

Please read it and make it better! :) 

## Publication date

This is part of the Octopus 2021 Q2 launch happening on 31-May but it can be published at any time.

### CODE REVIEW

File: blog/2021-06/asp-net-iis-on-prem-to-azure-app-service/index.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+First, review the **Update database** step. This script step uses a database connection string to connect to a database, so no changes are required here. We do need to ensure that our Octopus instance can connect to our Azure SQL database though. I need to update the Azure SQL database firewall to allow the Octopus Cloud [Static IP addresses](https://octopus.com/docs/octopus-cloud/static-ip) to connect to it.
+
+Next, we need to add a new **Azure App Service deployment** step. We've already configured our infrastructure so this isn't difficult. 
+
+![Azure App Service deployment step configuration](azure-web-app-deploy.png "width=500")
+
+This screenshot shows our step configuration. We're using a worker to deploy to Azure web app targets with the `web` role, and we specify the package, app service and connection string configuration updates. Octopus recently introduced an improved [Azure App Service deployment step](https://octopus.com/blog/azure-app-service-step) which allows us to wire-up our configuration updates directly in the step.
```
Comment (TeganAli) [inline blog/2021-06/asp-net-iis-on-prem-to-azure-app-service/index.md:115]: @robpearson would it make sense to update the last sentence in this para to:
Octopus recently introduced an improved [Azure App Service deployment step](https://octopus.com/blog/azure-app-service-step) which allows us to configure updates directly in the step.

Comment (robpearson) [reply blog/2021-06/asp-net-iis-on-prem-to-azure-app-service/index.md:115]: @TeganAli Yes, that suits and it's much shorter. :)

File: blog/2021-06/asp-net-iis-on-prem-to-azure-app-service/index.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
```
Comment (TeganAli) [inline blog/2021-06/asp-net-iis-on-prem-to-azure-app-service/index.md:115]:
```suggestion
This screenshot shows our step configuration. We're using a worker to deploy to Azure web app targets with the `web` role, and we specify the package, app service and connection string configuration updates. Octopus recently introduced an improved [Azure App Service deployment step](https://octopus.com/blog/azure-app-service-step) which allows us to configure updates directly in the step. 
```

File: blog/2021-06/asp-net-iis-on-prem-to-azure-app-service/index.md [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+![Migrate an ASP.NET web app from IIS on-premises to an Azure App Service](move-to-the-cloud-with-octopus-deploy-2021.png)
+
+Teams are migrating their applications and infrastructure to the cloud. The natural migration path for an ASP.NET web app running on Microsoft's IIS web server on-premises is to App Services on Microsoft's Azure cloud platform. It's relatively straightforward to change the way your application is hosted, but it's also important to ensure your CI/CD process is still repeatable and reliable. 
+
+In this blog post, I walk through how to update the deployment process of an existing web application with Octopus Deploy, moving it from on-premises to the cloud.
+
+## Example application
+
+We're using the [Random Quotes](https://github.com/octopussamples/randomquotes) web application as the example in this migration guide. This web application randomly retrieves and displays famous quotes. It's a simple application with an ASP.NET front end with a SQL Server back end to help illustrate the changes required without being overwhelming.
```
Comment (TeganAli) [inline blog/2021-06/asp-net-iis-on-prem-to-azure-app-service/index.md:22]:
```suggestion
We're using the [Random Quotes](https://github.com/octopussamples/randomquotes) web application as the example in this migration guide. This web application randomly retrieves and displays famous quotes. It's a simple application with an ASP.NET front end with a SQL Server back-end to help illustrate the changes required without being overwhelming. 
```

**Pull request merged**